packages/store-ui/src/Carousel/items.ts

```diff
@@ -0,0 +1,101 @@
+export default [
```

Comment:
Since this file is here only for storybook reasons, I'd rather move it to a `fixtures`/`stories` folder or something like this alongside the `.stores.ts` files. Also, I'd rather use `storecomponents` pictures. They have a free to use license